README.md

```diff
@@ -66,6 +66,7 @@ More about features and available dialogs you could read in [**the blog post**](
 - EF Core **5.0+** with projects under such target frameworks:
   - `net6.0`
   - `net5.0`
+  - `net7.0` (execute `dotnet tool update --global dotnet-ef --prerelease` first)
```

Comment:
I think instructions regarding preview EF Core installation is not needed here.

```suggestion
  - `net7.0` (preview)
```

Also please make proper order of target frameworks (7, 6, 5)